.github/workflows/wordpress-plugin-check.yml

```diff
@@ -0,0 +1,27 @@
+name: 'WordPress Plugin Check'
+on:
+  pull_request
+
+jobs:
+  check:
+    name: Plugin Check
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Install Composer dependencies
+      uses: ramsey/composer-install@v3
+      with:
+        composer-options: "--no-dev"
+
+    # Prepare the plugin as we prepare it in the ZIP creation
+    - name: Create openedx-commerce directory and copy files
+      run: |
+        mkdir openedx-commerce
+        rsync -av --exclude='*.git*' --exclude='.*' --exclude='/test/*' --exclude='/requirements/*' --exclude='/docs/*' --exclude='composer.lock' --exclude='*.yaml' --exclude='*.xml' --exclude='Makefile' ./ openedx-commerce/
```

Comment:
I don't like that we have a duplicate of the process in this line and this [action in a different workflow](https://github.com/openedx/openedx-wordpress-ecommerce/blob/main/.github/workflows/upload-release-zip.yml#L22-L27).

Do you think we could refactor both to use a `make release` target or something like this? What I really want is that we only have one source of truth when it comes to what this action tests and what the release zip finally contains

Reply:
I thought about it but wasn't sure where to implement it.

I'll implement that in the Makefile to use it in both. Thanks.